test-frame-test/src/test/java/io/skodjob/testframe/test/integration/KubeResourceManagerIT.java

```diff
@@ -7,58 +7,35 @@
 import io.fabric8.kubernetes.api.model.NamespaceBuilder;
 import io.skodjob.testframe.annotations.ResourceManager;
 import io.skodjob.testframe.annotations.TestVisualSeparator;
-import io.skodjob.testframe.clients.KubeClusterException;
 import io.skodjob.testframe.resources.KubeResourceManager;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestInstance;
 
 import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertThrows;
 
-@ResourceManager
+@ResourceManager(cleanResources = false)
 @TestVisualSeparator
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class KubeResourceManagerIT {
 
-    @BeforeAll
-    void setupAll() {
-        KubeResourceManager.getInstance().createResourceWithWait(
-                new NamespaceBuilder().withNewMetadata().withName("test").endMetadata().build());
-    }
-
     @BeforeEach
     void setupEach() {
         KubeResourceManager.getInstance().createResourceWithWait(
-                new NamespaceBuilder().withNewMetadata().withName("test2").endMetadata().build());
+                new NamespaceBuilder().withNewMetadata().withName("test").endMetadata().build());
     }
 
-    @AfterAll
-    void afterAll() {
-        assertNull(KubeResourceManager.getKubeClient().getClient().namespaces().withName("test2").get());
-        assertNull(KubeResourceManager.getKubeClient().getClient().namespaces().withName("test3").get());
+    @AfterEach
+    void afterEach() {
+        KubeResourceManager.getInstance().deleteResources();
```

Comment:
When the `cleanResources` is set to `false`, should we check in the AfterAll that the created namespace(s) are not removed?

Reply:
after each delete all namespaces `KubeResourceManager.getInstance().deleteResources()`, so I don't get your point

Reply:
Well, you would need to remove this line and then check it. Because for the `cleanResources=false` there is no tests or?

Reply:
added checks into afterEach, since after all is not needed for one test. I would say

Reply:
Yeah I added the comment here, but I meant it overall. Thanks